### ISSUE TITLE
Date filter remains when scrolling to next item in queue

### ISSUE THREAD

**Issue by sanlifer** [Neovici/cosmoz-omnitable#210]
![skarmavbild 2018-08-31 kl 10 25 41](https://user-images.githubusercontent.com/31737905/44901646-42641900-ad08-11e8-8ae0-34fb8761051e.png)

In /purchase/suppliers/list-queue, queue, Invoices tab.

1. Select a date in the invoice date column.
2. Scroll to next item in the queue.
3. Previous filter remains for each new item in the scroll.

**Comment by sanlifer** [comment]
@nomego Checked to see if this is still an issue. 
Tested in alpha v3.8.12, same procedure. I get the same result as described in Aug '18. So this is still an issue.

**Comment by megheaiulian** [comment]
@cristinecula Could we add first an acceptance test with the desired behavior ?

**Comment by cristinecula** [comment]
@sanlifer all filter columns are persisted across navigation, not just date columns. I understand that this is not the desired behavior?

@nomego 
`purchase-suppliers-view-core` uses `purchase-invoices-list-core` 
https://github.com/Neovici/cosmoz-frontend/blob/master/app/views/purchase/suppliers/view-core.html#L613-L616
`purchase-invoices-list-core` has `hash-param` set to `invoices-list-core-table`
https://github.com/Neovici/cosmoz-frontend/blob/master/app/views/purchase/invoices/list-core.html#L286

In other words, it's working as intended.

To achieve the expected behavior we could make use of a parameter on `purchase-invoices-list-core` that controls the `cosmoz-omnitable` `hash-param` value, and set it to `null`.

Also, this bug should be moved to cosmoz-frontend.
